notebook/static/tree/js/notebooklist.js

```diff
@@ -697,6 +707,73 @@ define([
         });
     };
 
+    NotebookList.prototype.move_selected = function() {
+        // TODO(nhdaly): Support moving multiple items at once.
+        if (this.selected.length !== 1){
+            return;
+        }
+
+        var that = this;
```

Comment:
Thanks Safia, nope that makes tons of sense now. Gotcha! Clever! Unnamed functions ... pretty crazy.

Thanks for taking the time to explain that to me! :)